lib/cyrusdb_quotalegacy.c

```diff
@@ -259,13 +259,8 @@ static int commit_subtxn(const char *fname, struct subtxn *tid)
         free(tid->fnamenew);
     } else if (tid->delete) {
         /* delete file */
-        r = xunlink(fname);
-        if (r == -1) {
-            xsyslog(LOG_ERR, "IOERROR: unlink failed",
-                             "fname=<%s>",
-                             fname);
```

Comment:
Same here

Reply:
This one is actually a meaningful behaviour change and not just a logging change, since we used to set `r = CYRUS_DBERROR` for ENOENT and with this change we'll ignore it.  I believe it's what we want though.

I guess this is another one where we could look for whether we ever log ENOENT here.  If it never happens in practice, then we never returned an error on this case, and perhaps the lack of an ENOENT exception was another oversight.  If we have been logging it, and haven't been annoyed by it, then maybe we do want it after all.  Perhaps we don't even use the quotalegacy db format, idk.